package.js

```diff
@@ -0,0 +1,21 @@
+var options = {
+  "version": "1.3.2",
+  "where": "client",
+  "packageName": "dvelopment:angular-validation"
+};
+
+// meta data
+Package.describe({
+  name: options.packageName,
+  version: options.version,
+  summary: 'Client-side Validation for AngularJS',
+  git: 'git@github.com:huei90/angular-validation.git',
```

Comment:
Should it be `https://github.com/huei90/angular-validation` instead of `git@github.com:huei90/angular-validation.git`? So, it will redirect to this github repo when clicking.